packages/cpp/src/browser/cpp-task-provider.spec.ts

```diff
@@ -63,7 +63,7 @@ class MockCppBuildConfigurationManager implements CppBuildConfigurationManager {
     ready: Promise<void> = Promise.resolve();
 }
 
-beforeEach(function() {
+beforeEach(function () {
```

Comment:
We really need to find a way to fix this formatting inconsistency between Theia and visual studio.

Reply:
sure, unfortunately a lot of the spec files currently have that spacing :(